magentoerpconnect/product_category.py

```diff
@@ -265,7 +250,7 @@ def parent_id(self, record):
         mag_cat_id = binder.to_openerp(record['parent_id'])
 
         if category_id is None:
-            raise MappingError("The product category with "
-                               "magento id %s is not imported." %
+            raise MappingError(_("The product category with "
+                                 "magento id %s is not imported.") %
```

Comment:
For info, in the lint check for Travis, it tests for error raised with translations.

magentoerpconnect/sale.py:359: [C8107(translation-required), SaleImportRule._rule_authorized] String parameter of raise "OrderImportRuleRetry" requires translation. Use _('The order has not been authorized.\nThe import will be retried later.')

Gonna ignore this case for technical errors.